src/Renderer.php

```diff
@@ -252,9 +254,9 @@ private function getTextblockStripeHeadings($stripe, $index) {
         $h2Count = 0;
         foreach ($headers as $header) {
             $tagName = $header->tagName;
-            $prefix = 'vssl-stripe--textblock--heading-';
+            $prefix = "vssl-stripe--textblock-$stripeId--heading";
             $headerIndex = $tagName === 'h1' ? $h1Count : $h2Count;
-            $id = $prefix . $index . '-' . $tagName . '-' . $headerIndex;
+            $id = $prefix . '-' . $tagName . '-' . $headerIndex;
```

Comment:
`$headingIndex` not `$headerIndex` here

This commit is beautiful

Reply:
Ah—and you should be able to grab the ID from the tag, right, rather than computing it again here?

Reply:
You're right! And that's better. I updated to have it pull the id from the DOMNode object directly.